test/common.py

```diff
@@ -221,7 +227,7 @@ def assertTensorsEqual(a, b):
                     self.assertTrue(torch.equal(nan_mask, b != b), message)
                     diff = a - b
                     diff[nan_mask] = 0
-                    if diff.is_signed():
+                    if diff.is_signed() and 'CharTensor' not in diff.type():
```

Comment:
we should just implement abs on CharTensor, right?  (Not necessary for this diff).

Reply:
Yes

Reply:
Can we mark this as a TODO/create an issue?